django/bossingest/views.py

```diff
@@ -231,6 +232,22 @@ def post(self, request):
         tile_size = ingest_config_data['ingest_job']['tile_size']
         database = ingest_config_data['database']
 
+        # Check that only permitted users are creating extra large ingests
+        large_ingest = 100000 * 100000 * 3000 # ~ 200 x 200 x 200 cubes
```

Comment:
Should there be something like `settings.INGEST_MAX_SIZE` instead of a hard coded value?